test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud", subset: ["template-runner.yml"])
+
+    task_runner = KubernetesDeploy::RunnerTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
+
+    task_runner.run(
+      task_template: 'hello-cloud-template-runner',
+      command: '/bin/bash',
+      args: ['rake', 'test']
+    )
+
+    assert_logs_match(/Starting task runner/)
```

Comment:
It isn't included since that's part of the logs. It can be retrieved through the logs.